content/en/product-handbook/testing/quality-assistance.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Quality Assistance"
+linkTitle: "Quality Assistance"
+weight: 1
+description: >
+  How the Quality Assistance process works on the team
```

Comment:
I would use "at Medic", rather than "on the team", so that people who are not from Medic and land on this page would understand right away to whom this process applies.

Reply:
Good call! I'll correct it.